src/client/restClient.ts

```diff
@@ -12,6 +12,12 @@ export class RestClient {
 	}
 
 	public get client(): JsonServiceClient {
-		return new JsonServiceClient(this.config.apiUrl);
+    const serviceClient = new JsonServiceClient(this.config.apiUrl);
+    if(this.config.isValid()) {
+      serviceClient.headers.set('X-CM-Cluster', this.config.cluster);
```

Comment:
DB clusterio ID siaip neturetu buti privalomas. Aplamai galima naudoti CM ir be DB. Todel cia turi buti optional. Kad ir naudojant, jei X-CM-Cluster neuzsetintas, tai ims default clusteri. 

Turi buti trys opcijos

1. Request veikia be DB cluster ID
2. DB cluster ID pasiima is config jei toks uzsetintas
3. DB clsuter galima perrasyti request metu. T.y. jei as confige turiu default, bet butent kazkoki request noriu atlikti konkreciai i kita cluster, tai galiu paduoti kito kluster id. 

3 variantas - DB cluster galetu pasiduoti kaip optional parametras.

Reply:
fixed, is esmes tai taip ir veikia, cia buvau pamirses idet checka ar configo cluster ID nera undefined. Bet jo, pirma ima is .env, jei jo nera - nenaudoja, ir galima overridint su kiekvienu requestu.